contracts/src/v0.7/dev/KeeperRegistrarDev.sol

```diff
@@ -87,16 +86,14 @@ contract KeeperRegistrarDev is TypeAndVersionInterface, ConfirmedOwner {
   event AutoApproveAllowedSenderSet(address indexed senderAddress, bool allowed);
 
   event ConfigChanged(
-    AutoApproveType configType,
-    uint32 windowSizeInBlocks,
-    uint16 allowedPerWindow,
+    AutoApproveType autoApproveConfigType,
+    uint32 autoApproveMaxAllowed,
     address keeperRegistry,
-    uint256 minLINKJuels
+    uint96 minLINKJuels
   );
 
-  constructor(address LINKAddress, uint256 minimumLINKJuels) ConfirmedOwner(msg.sender) {
+  constructor(address LINKAddress) ConfirmedOwner(msg.sender) {
```

Comment:
No real need to set minimumLINKJuels separately in constructor.

We'll need to set config alongwith registry address anyway

Reply:
on second thought we need this since register doesn't need the registry to be set